proposals/kubeflow-distributions.md

```diff
@@ -0,0 +1,70 @@
+
+## Objective
+
+Clarify how Kubeflow distributions will be owned and developed going forward.
+
+## Motivation
+
+Kubeflow can be divided into pieces
+
+  1. Individual Kubeflow applications (e.g. Pipelines, KFServing, notebooks, etc...)
+  1. Distributions of Kubeflow (e.g. Kubeflow on GCP, Kubeflow on AWS, MiniKF, etc...)
+
+
+ Since July, the Kubeflow community has been working on forming working groups to create greater
+ accountability for the different parts of Kubeflow.
+
+ At this point in time, Kubeflow has formed working groups with clear ownership for all of the individual Kubeflow
+ applications.
+
+ There is an ongoing debate about who should own and maintain Kubeflow distributions.
+
+ To date there are two categories of distributions
+
+  1. Kubeflow distributions tied to a specific platform (e.g. AWS, GCP, etc...)
+  1. Generic distributions (e.g. for MiniKube, any conformant K8s cluster, etc...)
+
+The former have been owned and maintained by the respective vendors. The general consensus is that these should continue
+to be owned and maintained by the respective vendors outside any KF working group.
+
+This leaves the question of what to do about generic distributions. In particular, in [kubeflow/community#402](https://github.com/kubeflow/community/pull/402) there was a long debate about whether the deployments working group would own them or not. That discussion appears to be converging with the decision being that the deployments working group will not own any distributions.
+
+## Proposal
+
+Going forward all distributions of Kubeflow should be owned and maintained outside of Kubeflow. 
+
+### What is a Kubeflow Distribution
+
+A Kubeflow distribution is an opinionated bundle of Kubeflow applications optimized for a particular use case or environment.
+
+### Ownership & Development
+
+Going forward new distributions of Kubeflow should be developed outside of the Kubeflow GitHub org. This ensures
```

Comment:
My issue is if this is completely outside the org, then what's the end user experience? A new user should be able to come to kubeflow/kubeflow and install a great experience (jupyter, kale, tf/pytorch job, katib, tfserving/seldon/triton/whatever).

Reply:
The install experience will be 

install pipelines

```
git clone git@github.com:kubeflow/manifests.git  git_manifests
kustomize build ./pipelines | kubectl apply -f
kustomize build ./notebooks | kubectl apply -f
....
```

In other words the install experience will be exactly what it is for everywhere Kubernetes application and similar to what it is for Tekton.

Reply:
@jlewi I am not quite sure it will be that simple, there will also be at least some cluster-level stuff which is required for all apps.

What the user chooses for the cluster-level stuff is probably going to affect the YAML for each of the sub-apps too.